english/Truth of the life of this world

```diff
@@ -0,0 +1,41 @@
+All living things and human beings have predetermined purposes to fulfil in life. These purposes owe their existence to the fact that everything has been created. An intelligent person notices that plan, design and wisdom exist in every detail of the infinitely varied world. This draws him to recognition of the Creator. He further concludes that since all living things are not the consequences of a random or mindless process they all serve important purposes. In the Qur'an, the last surviving authentic revealed guide to the true path for humanity, Allah repeatedly reminds us of the purpose of our life, which we tend to forget, and thereby summons us to clarity of mind and consciousness.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.